articles/communication-services/quickstarts/sms/includes/send-sms-java.md

```diff
@@ -45,6 +45,13 @@ Open your terminal or command window and navigate to the directory where you wou
 mvn archetype:generate -DgroupId=com.communication.quickstart -DartifactId=communication-quickstart -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false
 ```
 
+If you use Windows PowerShell, run instead:
```

Comment:
does it make sense to specify PowerShell version?

Reply:
It is not the place to ask these questions.